cpp/src/fil/fil.cu

```diff
@@ -154,7 +157,7 @@ struct forest {
     // Transform the output if necessary (sigmoid + thresholding if necessary).
     if (output_ != output_t::RAW) {
       transform_k<<<ceildiv(int(rows), FIL_TPB), FIL_TPB, 0, stream>>>(
-        preds, rows, output_ == output_t::CLASS, threshold_);
+        preds, rows, output_, ntrees_ > 0 ? 1.0f / ntrees_ : 1.0f, threshold_);
```

Comment:
parens around `1 / ntrees` not necessary but I think it'd be a bit easier to read